tokio/src/runtime/runtime.rs

```diff
@@ -382,7 +383,13 @@ impl Runtime {
     ///
     ///     // By entering the context, we tie `tokio::spawn` to this executor.
     ///     let _guard = rt.enter();
-    ///     function_that_spawns(s);
+    ///     let handle = function_that_spawns(s);
+    ///
+    ///     // When a `Runtime` instance is dropped, a shutdown signal will be
+    ///     // delivered to all tasks. So the spawned task may get canceled before
+    ///     // it gets to print the message. To force a deterministic output for this
+    ///     // example, we will wait until the task is finished.
```

Comment:
I don't think we need this comment. It makes sense for people that know of this bug, but I don't think people who see this fresh will need an explanation for the `block_on` call.
```suggestion
    ///     // Wait for the task before we end the test.
```